src/composables/useWeb3.ts

```diff
@@ -43,7 +43,7 @@ export default function useWeb3() {
   });
 
   const isMainnet = computed(() => {
-    return userNetwork.value.name === 'Mainnet';
+    return appNetwork.name === 'Mainnet';
```

Comment:
This function wasn't actually used so this change doesn't have much effect... I do think it should be checking for `appNetwork` instead of `userNetwork`.